src/main/java/blackjack/domain/betting/GameBettingManager.java

```diff
@@ -0,0 +1,77 @@
+package blackjack.domain.betting;
+
+import blackjack.domain.gamer.Dealer;
+import blackjack.domain.gamer.Player;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class GameBettingManager {
+
+    private static final Double WIN_BETTING_150 = 1.5;
+    private static final Double WIN_BETTING_100 = 1.0;
+    private static final Double TIE_BETTING = 0.0;
+    private static final Double LOSE_BETTING = -1.0;
+
+    private final Map<Player, Betting> playerBetting = new LinkedHashMap<>();
+
+    public GameBettingManager() {
+    }
+
+    public void registerPlayerBetting(Player player, Betting betting) {
+        playerBetting.put(player, betting);
+    }
+
+    public void calculatePlayerProfit(Dealer dealer, Player player) {
+        double profitRate = decideWinner(dealer, player);
+        playerBetting.put(player, playerBetting.get(player).calculateBettingMoney(profitRate));
+    }
+
+    public double getDealerResult() {
+        return playerBetting.values().stream()
+                .map(Betting::getBettingMoney)
+                .mapToDouble(bettingResult -> -bettingResult)
+                .sum();
+    }
+
+    public Map<Player, Betting> getPlayersResult() {
+        return playerBetting;
+    }
+
+    private double decideWinner(Dealer dealer, Player player) {
+        if (playerLoses(dealer, player)) {
+            return LOSE_BETTING;
+        }
+        if (playerWinsWithBlackJack(dealer, player)) {
+            return WIN_BETTING_150;
+        }
+        if (playerWinsWithoutBlackJack(dealer, player)) {
+            return WIN_BETTING_100;
+        }
+        return TIE_BETTING;
+    }
+
+    private boolean playerWinsWithBlackJack(Dealer dealer, Player player) {
+        return player.isBlackJack() && !dealer.isBlackJack();
+    }
+
+    private boolean playerWinsWithoutBlackJack(Dealer dealer, Player player) {
+        int dealerScore = dealer.calculateTotalScore();
+        int playerScore = player.calculateTotalScore();
+
+        if (!player.isBusted() && dealer.isBusted()) {
+            return true;
+        }
+        return (!player.isBusted() && playerScore > dealerScore);
+    }
+
+    private boolean playerLoses(Dealer dealer, Player player) {
+        int dealerScore = dealer.calculateTotalScore();
+        int playerScore = player.calculateTotalScore();
+
+        if (player.isBusted()) {
+            return true;
+        }
+        return (!dealer.isBusted() && playerScore < dealerScore);
```

Comment:
```suggestion
        if (player.isBusted()) {
            return true;
        }
        if (dealer.isBusted()) {
            return false;
        }
        return playerScore < dealerScore;
```
(반영하지 않으셔도 됩니다.) 개인적으로 &&절은 조건이 조금만 복잡해도(!를 쓰는 등과 같이) 가독성이 떨어진다고 생각해요. 현재의 조건을 생각해보면

```
player가 지는 조건
1. player가 bust면 진다(true).
2. dealer가 bust면 지는 것이 아니다(false).
3. player가 dealer보다 점수가 작으면 진다(true).
```
라고 보면 if로 분기를 태우는 것이 더 가독성이 높은 것 아닐까싶어요.